packages/cli/test/helpers/parse-table.ts

```diff
@@ -1,4 +1,4 @@
-export function getDataFromIntro(output: string): {
```

Comment:
I think this should be even more specific to what it does. It looks for `Deployments for`, for example. What about `pluckIdentifiersFromDeploymentList`? (I can't think of a great name for this.)

Reply:
It's a little long but I think it's great